GoogleForCreators/web-stories-wp
# Show link tooltip only on hover (first pass with old design)

## Summary



First pass with old design.



## User-facing changes

Link tooltip now appears on hover only (when the element is selected it will be hidden).

## Testing Instructions

Select an element and create a link.
It should only appear on hover (when element is not selected).

---



Fixes #1968
